src/personal_api_keys/NewPersonalApiKey.tsx

```diff
@@ -0,0 +1,52 @@
+import React ,{ useEffect, useState } from "react";
+import MDSpinner from "react-md-spinner";
+import {PersonalApiKeyForm} from "./PersonalApiKeyForm";
+import { DataRetrievalState} from '../types/retrievingDataTypes';
+
+
+export const NewPersonalApiKey: React.FC = () => {
+  const [allPersonalApiKeys, setAllPersonalApiKeys] = useState([]);
+  const [allPersonalApiKeysFetching, setallPersonalApiKeysFetching] = useState<DataRetrievalState>("NEVER_DID_RETRIEVE");
+
+  useEffect(() => {
+    setallPersonalApiKeysFetching("RETRIEVING");
+    fetch('/api/v4/personalapikeys')
+    .then(response=>{
+      if (response.status === 200) {
+        return response.json();
+      } else {
+          // todo fix this erro notification
+          // props.addNotification(status, 2000);
```

Comment:
do you have a ticket to fix this error notification?

Reply:
Thanks for reminding me, I created this one:
https://github.com/nens/lizard-management-client/issues/598